package.json

```diff
@@ -6,17 +6,45 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "jest"
```

Comment:
once prettier is added let's add one more script:

```
"format": "prettier --write .",
```

Reply:
fixed in commit 41f3fe9644339bc8d444d2dff689b46fd1d9e35d